test/skate_web/controllers/detours_controller_test.exs

```diff
@@ -45,6 +49,57 @@ defmodule SkateWeb.DetoursControllerTest do
       assert %{"data" => %{"missed_stops" => [^expected_missed_stop]}} = json_response(conn, 200)
     end
 
+    @tag :authenticated
+    test "returns connection points", %{conn: conn} do
```

Comment:
It sort of worries me that there's no test validating that this works when `connection_start`/`connection_end` are `nil`, but I did test it by hand (at least for `connection_end`) and it worked just fine.... and also the test itself could be kind of annoyingly repetitive. 

I'm kind of on the fence about this whether I would even add a test for this, because of ☝️ points, so I'm definitely not going to block approval on it! 😅

Reply:
Going to followup with another PR to add test coverage for this